docs/modules/user-guide/partials/proc_adding-components-to-a-devfile.adoc

```diff
@@ -9,54 +9,69 @@ Each component in a single devfile must have a unique name and use one of the ob
 |===
 |Key |Type| Required| Description
 
-|container
-|componentObject
+|`container`
+|`componentObject`
 |no
-|The list of containers that you use.
+|The list of containers.
+
+|`kubernetes`
+|`componentObject`
+|no
+|The Kubernetes cluster.
+
+|`openshift`
+|`componentObject`
+|no
+|The OpenShift container.
+
+|`volume`
+|`componentObject`
+|no
+|The list of volume components.
 |===
 
 .container object
```

Comment:
Something else to consider using the monospaced font on (this may apply to several other table headings on this page):
```suggestion
.`container` object
```
:) I have no idea if Antora will render it as expected, so watch out if you do this.